lint

```diff
@@ -8,10 +8,14 @@
   "license": "MIT",
   "devDependencies": {
     "jest": "^23.4.1",
-    "prettier": "1.13.7"
+    "prettier": "^1.13.7",
+    "prettier-check": "^2.0.0"
```

Comment:
Can you add this via `yarn add --dev prettier-check`, so it updates `yarn.lock` as well?